src/controllers/feedController.js

```diff
@@ -0,0 +1,94 @@
+const Feed = require("../models/feed");
+const cloudinary = require("../utils/cloudinary");
+const mongoose = require("mongoose");
+
+exports.createFeed = async (req, res) => {
+  const { createdBy, text } = req.body;
+  try {
+    let result;
+    if (req.file) {
+      result = await cloudinary.uploader.upload(req.file.path);
+    }
+    //instance of post
+    const data = {
+      createdBy: createdBy,
+      text: text,
+      status: "active",
+      imgLink: result?.secure_url || "",
+      cloudinaryId: result?.public_id || "",
+    };
+    let feed = new Feed(data);
+    //saving post
+    await feed.save();
```

Comment:
What if, save was not successful?

Reply:
in next line  there is error catching code for save